components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -339,6 +357,7 @@ - (void)updateDrawerState:(CGFloat)transitionPercentage {
 - (void)viewDidLoad {
   [super viewDidLoad];
 
+  self.initialDrawerFactor = [self shouldPresentFullScreen] ? 1 : (CGFloat)0.5;
```

Comment:
this isn't needed i think. The getter makes sure it's 1 if its full screen, and 0.5 is the init value.